spec/index.bs

```diff
@@ -4372,6 +4407,7 @@ The {{GPUTextureUsage}} flags determine how a {{GPUTexture}} may be used after i
                 1. Set |t|.{{GPUTexture/dimension}} to |descriptor|.{{GPUTextureDescriptor/dimension}}.
                 1. Set |t|.{{GPUTexture/format}} to |descriptor|.{{GPUTextureDescriptor/format}}.
                 1. Set |t|.{{GPUTexture/usage}} to |descriptor|.{{GPUTextureDescriptor/usage}}.
+                1. Set |t|.{{GPUTexture/textureBindingViewDimension}} to |descriptor|.{{GPUTextureDescriptor/textureBindingViewDimension}}.
```

Comment:
If we are going to ignore this validation on Core, then on Core we should also not reflect it - i.e. even if a `textureBindingViewDimension` is requested we should expose `undefined`.

Sorry, apparently this comment vanished into the void somehow (yet was still saved as a draft on my computer). This addresses your question on the other thread https://github.com/gpuweb/gpuweb/pull/5257#discussion_r2299146644